cmd/legacy/publish/publish.go

```diff
@@ -17,11 +18,7 @@ var PublishCmd = &cobra.Command{
 	Short: "Publishes an app to the public app gallery",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		err := publish(gql.GetClient())
-		if err != nil {
-			return err
-		}
-
-		return nil
+		return errorhandling.ErrorAlreadyPrinted(err)
```

Comment:
Can we just decide on one way do it? 😄 
option 1.
```golang
err := someFn(...)
return errorhandling.ErrorAlreadyPrinted(err)
```
option 2.
```golang
return errorhandling.ErrorAlreadyPrinted(someFn(...))
```

Reply:
I rebased and made the changes for option 1 :)